mrp_multi_level/views/mrp_area_views.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record model="ir.ui.view" id="mrp_area_list">
+        <field name="name">mrp.area.list</field>
+        <field name="model">mrp.area</field>
+        <field name="type">list</field>
+        <field name="arch" type="xml">
+            <list>
+                <field name="name" />
+                <field name="warehouse_id" />
+                <field name="company_id" groups="base.group_multi_company" />
+                <field name="location_id" />
+                <field name="calendar_id" />
+            </list>
+        </field>
+    </record>
+    <record model="ir.ui.view" id="mrp_area_form">
+        <field name="name">mrp.area.form</field>
+        <field name="model">mrp.area</field>
+        <field name="type">form</field>
+        <field name="arch" type="xml">
+            <form string="MRP Area">
+                <sheet>
+                    <div class="oe_button_box" name="button_box" />
+                    <widget
+                        name="web_ribbon"
+                        title="Archived"
+                        bg_color="bg-danger"
+                        invisible="active"
+                    />
+                    <label for="name" class="oe_edit_only" />
+                    <h1>
+                        <field name="name" />
+                    </h1>
+                    <group colspan="4" col="2">
+                        <group>
+                            <field name="active" invisible="1" />
+                            <field name="warehouse_id" />
```

Comment:
Hi Jasmin,

We could adjust this to avoid a small bug that occurs when clicking "New" on the warehouse_id field if the Storage Locations setting is not active. Let me know what you think! 😊

![Peek 2024-12-05 16-50](https://github.com/user-attachments/assets/7d633770-38d6-49b0-9d76-3090f170dad9)


```suggestion
                            <field name="warehouse_id"
                                   options="{'no_create': True, 'no_create_edit': True}"
                                   force_save="1"/>
```

Reply:
I did a separate PR about that small issue: https://github.com/OCA/manufacture/pull/1424. Maybe it's better to keep it separated from the migration.